app/utils/exceptions.py

```diff
@@ -0,0 +1,37 @@
+import typing as tp
+
+
+class LimitOffsetError(Exception):
```

Comment:
all of your exceptions are the same. refactor to remove duplication

Reply:
first of all its i can see what exception i will raise and i will add logging for detecting error messages and in what part of code this was raised;I dont know if it is necessary, if not, I can remove it

Reply:
I mean having different exception types for different situations is good. on the other hand you can reduce duplication by having base exception class
```
class BaseException(Exception):
	    def __init__(self, message: tp.Union[str, tp.Dict[str, str]]) -> None:
        	self.message = message
class LimitOffsetError(BaseException):
	pass
class SortException(BaseException):
	pass
```

Reply:
yep, done